<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserWindow</class>
 <widget class="QMainWindow" name="BrowserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manitou</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/Resources.qrc">
    <normaloff>:/icons/manitou.png</normaloff>:/icons/manitou.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionInspector"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLocation"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuFind">
     <property name="title">
      <string>&amp;Find</string>
     </property>
     <addaction name="actionSearch"/>
    </widget>
    <widget class="QMenu" name="menuHighlight">
     <property name="title">
      <string>Highlight</string>
     </property>
     <addaction name="actionLinks"/>
     <addaction name="actionParagraphs"/>
    </widget>
    <addaction name="menuFind"/>
    <addaction name="menuHighlight"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionToolbar"/>
    <addaction name="separator"/>
    <addaction name="actionReload"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionSource"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionInspector">
   <property name="text">
    <string>Web &amp;Inspector</string>
   </property>
  </action>
  <action name="actionLocation">
   <property name="text">
    <string>Open Location...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Google Search...</string>
   </property>
   <property name="shortcut">
    <string>Meta+K</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="text">
    <string>Hide &amp;Toolbar</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>&amp;Reload Page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionLinks">
   <property name="text">
    <string>Links</string>
   </property>
  </action>
  <action name="actionSource">
   <property name="text">
    <string>View Source</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionParagraphs">
   <property name="text">
    <string>Paragraphs</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
